intelmq/bots/outputs/elastic/output.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+import sys
+from json import loads
+from collections.abc import Mapping
+
+from elasticsearch import Elasticsearch
+
+from intelmq.lib.bot import Bot
+
+#
+# TODO: recursion depth check
```

Comment:
Do you mean checking for loops? That will never happen (and if, it should be prohibited by the core).